docs/scripts/SetupBurningOKR_windows.bat

```diff
@@ -1,20 +1,82 @@
 @ECHO OFF
-cls
+CLS
+
 ECHO #### Cloning project ####
 echo.
-git clone https://github.com/BurningOKR/BurningOKR
+git clone https://github.comBurningOKR/BurningOKR
 echo.
-ECHO #### Installing Angular/CLI ####
+
+set p c=Is Node.js LTS already installed [yN]
+if i %c% == Y GOTO :install_angular
+if i %c% == N GOTO :install_nodejs
+GOTO :install_nodejs
+
+:install_nodejs
+ECHO #### Installing Node.js LTS v16.14.2 64bit ####
+powershell -Command (New-Object Net.WebClient).DownloadFile('https://nodejs.org/dist/v16.14.2/node-v16.14.2-x64.msi', 'node-v16.14.2-x64.msi')
```

Comment:
Es köönte sinnvoll sein, die Node-Version in eine Variable zu schreiben, damit diese sich später leichter ändern lässt